src/main/kotlin/com/workos/fga/types/QueryOptions.kt

```diff
@@ -0,0 +1,36 @@
+package com.workos.fga.types
+
+import com.fasterxml.jackson.annotation.JsonInclude
+import com.fasterxml.jackson.annotation.JsonProperty
+import com.workos.common.models.Order
+
+@JsonInclude(JsonInclude.Include.NON_NULL)
+class QueryOptions @JvmOverloads constructor(
```

Comment:
This class should include a `warrantToken` attribute just like `CheckRequestOptions` does, right?

Reply:
Nvm, I misunderstood what `CheckRequestOptions` was. Leaving a separate comment on the `query` method instead.